lib/sitemap.xml

```diff
@@ -37,7 +37,7 @@
     </url>
   {% endunless %}{% endfor %}
   {% endunless %}{% endfor %}
-  {% for file in site.html_files %}
+  {% for file in site.sitemap_files %}
```

Comment:
I'll just move it to `page` scope instead of `site`. I don't know why I didn't think of this before.